### DISCUSSION TITLE
Branch management

### DISCUSSION THREAD

**Discussion by milivojm** [CntoDev/level-up#87] [Ideas]
Considering branches are connected with promotion/demotion of specialists, I'd like to discuss about branch management. Few questions that come out to mind:

1. Do we make this as separate service? Core Roster is big enough and putting more stuff in it will make it a happy monolith.
2. What do we include in Branch Management then? Corporals could see who is in their branch. Anything else?

**Comment by enrico-ghidoni** [comment]
IMO this should definitely be a separate service although as far as I know there is no real purpose other than promotions/demotions for handling branches. That is with the exclusion of Interviewing branch and Member Management branch, which should alter the permissions for branch members.
